smithy-swift-codegen/src/test/kotlin/TestUtils.kt

```diff
@@ -344,7 +344,7 @@ fun getSettingsNode(
         .withMember("sdkId", Node.from(sdkId))
         .withMember("author", Node.from("Amazon Web Services"))
         .withMember("gitRepo", Node.from("https://github.com/aws-amplify/amplify-codegen.git"))
-        .withMember("swiftVersion", Node.from("5.5.0"))
+        .withMember("swiftVersion", Node.from("5.7.0"))
```

Comment:
lol idk what this file does

Reply:
Looks like its part of settings used in tests

Reply:
I am going to revert this since it seems related to swift-tools-version